src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,29 @@
+.calendar {
+  width: $calendar-width;
+  height: $calendar-height;
```

Comment:
you don't real need to set calendar `height` - it should be determined by its content